tools/IoTEdgeDevOps/DevOpsLib/ReleaseManagement.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace DevOpsLib
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using DevOpsLib.VstsModels;
+    using Flurl;
+    using Flurl.Http;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Linq;
+
+
+    public class ReleaseManagement
+    {
+        const string ReleasePathSegmentFormat = "{0}/{1}/_apis/release/releases";
+
+        readonly DevOpsAccessSetting accessSetting;
+
+        public ReleaseManagement(DevOpsAccessSetting accessSetting)
+        {
+            this.accessSetting = accessSetting;
+        }
+
+        /// <summary>
+        /// This method is used to get latest release result of given release definition Id and branch name with descending order.
+        /// Reference: https://docs.microsoft.com/en-us/rest/api/azure/devops/release/releases/list?view=azure-devops-rest-5.1
+        /// </summary>
+        /// <param name="definitionId">release definition Ids</param>
+        /// <param name="branchName">github repository branch name</param>
+        /// <returns>List of IoT Edge releases</returns>
+        public async Task<List<IoTEdgeRelease>> GetReleasesAsync(ReleaseDefinitionId definitionId, string branchName, int top = 5)
+        {
+            ValidationUtil.ThrowIfNullOrWhiteSpace(branchName, nameof(branchName));
```

Comment:
Is there any reason the `definitionId` is not being checked here?

Reply:
it is an enum; don't need to check.